### PR TITLE
Add CN doc for linalg.solve

### PR DESCRIPTION
**Pull request by veyron95**
Add zh-cn documentation for paddle.linalg.solve API.

pre-view picture

图片

### CODE REVIEW

**Review by JiabinYang — COMMENTED** [review]
Post pre-view picture in your PR message

File: docs/api/paddle/solve_cn.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.. _cn_api_tensor_solve:
```
Comment (TCChenlong) [inline docs/api/paddle/solve_cn.rst]: 目录和名称错了，应该都是 paddle.linalg.solve

File: docs/api/paddle/Overview_cn.rst [hunk 2]
```diff
@@ -248,6 +248,7 @@ tensor线性代数相关
     " :ref:`paddle.tril <cn_api_tensor_tril>` ", "返回输入矩阵 input 的下三角部分，其余部分被设为0"
     " :ref:`paddle.triu <cn_api_tensor_triu>` ", "返回输入矩阵 input 的上三角部分，其余部分被设为0"
     " :ref:`paddle.multi_dot<cn_api_tensor_multi_dot>` ", "计算多个矩阵相乘"
+    " :ref:`paddle.solve <cn_api_tensor_solve>` ", "求线性方程组 Ax=B 的解"
```
Comment (TCChenlong) [inline docs/api/paddle/Overview_cn.rst]: 这行删除即可

File: docs/api/paddle/solve_cn.rst (hunk 1 above)
Comment (veyron95) [reply docs/api/paddle/solve_cn.rst]: Done

File: docs/api/paddle/Overview_cn.rst (hunk 2 above)
Comment (veyron95) [reply docs/api/paddle/Overview_cn.rst]: Done

**Review by TCChenlong — APPROVED** [review]

**Pull request merged**